nautobot-app-chatops/{{ cookiecutter.project_slug }}/pyproject.toml

```diff
@@ -144,17 +121,22 @@ ignore = [
     "D401",  # First line of docstring should be in imperative mood
     "D407",  # Missing dashed underline after section
     "D416",  # Section name ends in colon
+    "E501",  # Line too long
 ]
 
 [tool.ruff.lint.pydocstyle]
 convention = "google"
 
-[tool.ruff.per-file-ignores]
+[tool.ruff.lint.per-file-ignores]
 "{{ cookiecutter.app_name }}/migrations/*" = [
-    "D",  # pydocstyle
+    "D",
+    "F",
+    "E",
+    "W"
```

Comment:
Seems weird to me to disable flake8 rules against migrations - why?

Reply:
cc.: @Kircheneer

Reply:
Agreed, can undo